doc/nrf/libraries/bluetooth_services/services/fast_pair.rst

```diff
@@ -193,6 +196,9 @@ The FMDN extension implementation also acts as middleware between the user appli
 The DULT module integration is required for small and not easily discoverable accessories.
 The :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT` is enabled by default.
 
+The :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_MOTION_DETECTOR` of the FMDN extension selects the :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR` Kconfig option to enable the motion detector feature of the DULT module.
+With this option enabled, the FMDN extension passes the DULT motion detector callbacks from the DULT module to application.
```

Comment:
refer to the DULT library guide:

```suggestion
With this option enabled, the FMDN extension passes the DULT motion detector callbacks from the DULT module to application.
To learn about implementation details for the DULT motion detector, see (...).
```

Reply:
This is invalid as you didn't provide the implementation details from my suggestion:

https://github.com/nrfconnect/sdk-nrf/pull/18138#discussion_r1816346432

Reply:
I guess it won't hurt to provide a link for further reading here.